plugins/extractit/plugin.spec.yaml

```diff
@@ -4,10 +4,11 @@ products: ["insightconnect"]
 name: extractit
 title: ExtractIt
 description: The ExtractIt plugin has a collection of actions used to extract various information from text. These include URLs, domains, emails, IPs, and more
-version: 2.1.2
+version: 2.2.0
 vendor: rapid7
 support: rapid7
 status: []
+cloud_ready: true
```

Comment:
@pmara-r7 Making an exception for this one since it doesn't have a connection and is now in Python - which is the thing that was preventing it from running in the cloud.

Reply:
perfect! Thanks